language-support/ts/eslint.bzl

```diff
@@ -0,0 +1,30 @@
+load("@language_support_ts_deps//eslint:index.bzl", _eslint_test = "eslint_test")
+
+def eslint_test(name, srcs, tsconfig = ":tsconfig.json", package_json = ":package.json", data = [], **kwargs):
+    """Run eslint on the given typescript source.
+
+    Args:
+      name: The name of the generated test rule.
+      srcs: The source files to lint.
+      tsconfig: The tsconfig.json file.
+      package_json: The package.json file.
+      data: Additional runtime dependencies.
+    """
+    eslint_deps = [
+        "@language_support_ts_deps//@typescript-eslint",
+        "@language_support_ts_deps//tsutils",
+    ]
+    templated_args = [
+        "--ext",
+        "ts",
+        "--config",
+        "$(rlocation $(location %s))" % package_json,
+        '--parser-options={"tsconfigRootDir":"$(dirname $(rlocation $(location %s)))"}' % tsconfig,
```

Comment:
`tsconfigRootDir` is set so that the eslint plugin finds the `tsconfig.json` file as specified in `package.json`.